scopes/defender/tester/tester.main.runtime.ts

```diff
@@ -166,19 +166,21 @@ export class TesterMain {
   async getTestsResults(
     component: IComponent,
     idHasVersion = true
-  ): Promise<{ testsResults?: TestsResult; loading: boolean } | undefined> {
+  ): Promise<{ testsResults?: TestsResult; loading: boolean, coverage?: unknown[] } | undefined> {
```

Comment:
can we cast it to a proper type instead of unknown? We should know the type here?